public/js/googleAuth.js

```diff
@@ -0,0 +1,26 @@
+function handleCredentialResponseRegister(response) {
```

Comment:
did you make provision for people that have signed up the normal way before but wants to use google auth to login

Reply:
I don't understand